### ISSUE TITLE
need to turn off logging

### ISSUE THREAD

**Issue by doutriaux1** [CDAT/cdat_info#2]
```
-------------------- >> begin captured logging << --------------------
urllib3.connectionpool: DEBUG: Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
urllib3.connectionpool: DEBUG: Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
urllib3.connectionpool: DEBUG: Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
urllib3.connectionpool: DEBUG: Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
urllib3.connectionpool: DEBUG: Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
urllib3.connectionpool: DEBUG: Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
urllib3.connectionpool: DEBUG: Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
urllib3.connectionpool: DEBUG: Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
urllib3.connectionpool: DEBUG: Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
urllib3.connectionpool: DEBUG: Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
urllib3.connectionpool: DEBUG: Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
urllib3.connectionpool: DEBUG: Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
urllib3.connectionpool: DEBUG: Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
urllib3.connectionpool: DEBUG: Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
urllib3.connectionpool: DEBUG: Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
urllib3.connectionpool: DEBUG: Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
urllib3.connectionpool: DEBUG: Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
urllib3.connectionpool: DEBUG: Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
urllib3.connectionpool: DEBUG: Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
urllib3.connectionpool: DEBUG: Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
urllib3.connectionpool: DEBUG: Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
urllib3.connectionpool: DEBUG: Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
urllib3.connectionpool: DEBUG: Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
urllib3.connectionpool: DEBUG: Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
urllib3.connectionpool: DEBUG: Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
urllib3.connectionpool: DEBUG: Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
urllib3.connectionpool: DEBUG: https://uvcdat-usage.llnl.gov:443 "POST /log/add/ HTTP/1.1" 503 412
urllib3.connectionpool: DEBUG: https://uvcdat-usage.llnl.gov:443 "POST /log/add/ HTTP/1.1" 503 412
urllib3.connectionpool: DEBUG: Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
urllib3.connectionpool: DEBUG: Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
urllib3.connectionpool: DEBUG: Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
urllib3.connectionpool: DEBUG: Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
urllib3.connectionpool: DEBUG: Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
urllib3.connectionpool: DEBUG: Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
urllib3.connectionpool: DEBUG: Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
urllib3.connectionpool: DEBUG: Starting new HTTPS connection (1): uvcdat-usage.llnl.gov
--------------------- >> end captured logging << ---------------------
```

**Comment by doutriaux1** [comment]
see: https://stackoverflow.com/questions/11029717/how-do-i-disable-log-messages-from-the-requests-library

**Comment by doutriaux1** [comment]
fixed by #3
